### ISSUE TITLE
Clean up and improve minc_deletion.pl

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris-MRI#402]
### Description

Now that we will have an amazing script to delete an upload in the tools directory, it would be good to improve and debug the `minc_deletion.pl` script to reflect the behaviour of the upload deletion script.

**_Known bugs so far:_**
- does not deal with files that have processed data attached to it
- delete one item at a time instead of all them at once so that when one fails, we lost partially data in the MRI tables
- not sure that it is dealing properly with the violations tables properly (if can delete a file based on its seriesUID, then we should be able to delete that file from the violations table if it is in one of those tables)
- only update the count of number of minc inserted if the deletion worked [(redmine #13842)](https://redmine.cbrain.mcgill.ca/issues/13842)

**Comment by cmadjar** [comment]
The only two things missing from the new script would be:
- being able to delete a scan based on its SeriesUID
- update the count of number of MINC inserted if the deletion worked (redmine #13842)

Once this is done, then we can remove the `minc_deletion.pl` script (after having checked that no other scripts are calling it)

**Comment by cmadjar** [comment]
I think everything has been taken care of in that ticket. @nicolasbrossard feel free to create a new issue if something is missing.
